### ISSUE TITLE
OperationalError: Unknown column 'pootle_store_store.pootle_path' in 'where clause

### ISSUE THREAD

**Issue by julen** [translate/pootle#4088]
```python
Traceback:
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  111.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "~/dev/translate/pootle/pootle/core/decorators.py" in wrapped
  116.         return func(request, path_obj, *args, **kwargs)
File "~/dev/translate/pootle/pootle/core/decorators.py" in _wrapped
  325.             return func(request, *args, **kwargs)
File "~/dev/translate/pootle/pootle/apps/pootle_language/views.py" in export_view
  90.     ctx = get_export_view_context(request)
File "~/dev/translate/pootle/pootle/core/helpers.py" in get_export_view_context
  125.     unit_total_count = units.count()
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/django/db/models/query.py" in count
  338.         return self.query.get_count(using=self.db)
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/django/db/models/sql/query.py" in get_count
  436.         number = obj.get_aggregation(using=using)[None]
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/django/db/models/sql/query.py" in get_aggregation
  402.         result = query.get_compiler(using).execute_sql(SINGLE)
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/django/db/models/sql/compiler.py" in execute_sql
  787.             cursor.execute(sql, params)
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/django/db/backends/utils.py" in execute
  81.             return super(CursorDebugWrapper, self).execute(sql, params)
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/django/db/backends/utils.py" in execute
  65.                 return self.cursor.execute(sql, params)
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/django/db/utils.py" in __exit__
  94.                 six.reraise(dj_exc_type, dj_exc_value, traceback)
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/django/db/backends/utils.py" in execute
  65.                 return self.cursor.execute(sql, params)
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/django/db/backends/mysql/base.py" in execute
  129.             return self.cursor.execute(query, args)
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/MySQLdb/cursors.py" in execute
  205.             self.errorhandler(self, exc, value)
File "~/.virtualenvs/pootle/lib/python2.7/site-packages/MySQLdb/connections.py" in defaulterrorhandler
  36.     raise errorclass, errorvalue
```

This has to be some recent regression, as MLO, which is at 324684b, doesn't exhibit it.

**Comment by julen** [comment]
`git bisect` points to edb73dcb49 as the cause of the regression.

**Comment by julen** [comment]
This happens for admin users only: their filter now omits `store__translation_project__project__disabled=False`, which makes Django's ORM to fail.

**Comment by ta2-1** [comment]
There is the same error on current master https://github.com/translate/pootle/commit/7672b47266d1fc747321d2405b34edf44096b838 but it should be addressed in https://github.com/translate/pootle/pull/4317 asap.

**Comment by julen** [comment]
With the changes merged from #4317 I can confirm is now fixed.
